src/CurrentBlockchainStatus.cpp

```diff
@@ -137,6 +137,11 @@ CurrentBlockchainStatus::calculate_emission_in_blocks(
 
         uint64_t coinbase_amount = get_outs_money_amount(blk.miner_tx);
 
+        if(start_blk == 0 && coinbase_amount == 8800000000000000)
```

Comment:
It's not necessary to have the `&&` clause, as either of these conditions is sufficient, but I used both for verbosity.

Reply:
Can you subtract `8700051446427001` instead? We want to be exact.

Reply:
This has been fixed by changing `coinbase_amount` to `99948553572999` on height 0.